docs/source/performance/cache-backends.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring cache backends
+description: How to configure Apollo Server's cache
+---
+
+> ⚠️ New in Apollo Server 3.9: we **highly recommend** that all users pass `cache: "bounded"` or configure their cache in a manner which isn't unbounded (which is current default behavior) to protect your server from attacks which exhaust available memory causing a DOS. Continue reading for more information on configuring Apollo Server's cache.
```

Comment:
is it reasonable to suggest disabling APQ as another approach here? or can this link to a place that has a bit more details about the issue that could mention that?

Reply:
I linked to the section immediately below which goes in to more details including APQ cache